crane/container.go

```diff
@@ -40,7 +40,7 @@ type ContainerInfo interface {
 	ActualName() string
 	Image() string
 	ID() string
-	Dependencies() *Dependencies
+	Dependencies(excluded []string) *Dependencies
```

Comment:
I know the global is ugly, but I didn't find a better solution. Basically all methods can't have arguments if you want to use them easily from a template.